pkg/pillar/cmd/zedrouter/zedrouter.go

```diff
@@ -303,6 +313,13 @@ func Run() {
 
 	updateLispConfiglets(&zedrouterCtx, zedrouterCtx.legacyDataPlane)
 
+	// XXX for flowstats
```

Comment:
Does XXX imply something remains to be done?

Reply:
No. removed.